docs/Guide/CLI/introduction.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Introduction
+sidebar_position: 0
+---
+
+# Introduction
+
+This guide will teach you things such as:
+
+- Installing Sapphire CLI
+- Creating a bot using Sapphire CLI
+- Creating components using Sapphire CLI
+- Creating custom component templates to use with Sapphire CLI
```

Comment:
Make these links to their respective pages

Installing -> Getting Started + hash route to the right header
Creating a bot -> Getting started + hash route to the right header
etc

See https://docusaurus.io/docs/versioning#linking-docs for how to do that.